ansible/vars/enikshay/enikshay_public.yml

```diff
@@ -106,28 +106,25 @@ formplayer_db_name: formplayer
 postgresql_dbs:
   - django_alias: default
     name: "{{localsettings.PG_DATABASE_NAME}}"
-#  - django_alias: proxy
-#    name: commcarehq_proxy
-#  - django_alias: p1
-#    name: commcarehq_p1
-#    shards: [0, 204]
-#  - django_alias: p2
-#    name: commcarehq_p2
-#    shards: [205, 409]
-#  - django_alias: p3
-#    name: commcarehq_p3
-#    shards: [410, 614]
-#  - django_alias: p4
-#    name: commcarehq_p4
-#    shards: [615, 819]
-#  - django_alias: p5
-#    name: commcarehq_p5
-#    shards: [820, 1023]
+  - django_alias: proxy
+    name: commcarehq_proxy
+  - django_alias: p1
+    name: commcarehq_p1
+    shards: [0, 204]
+  - django_alias: p2
+    name: commcarehq_p2
+    shards: [205, 409]
+  - django_alias: p3
+    name: commcarehq_p3
+    shards: [410, 614]
+  - django_alias: p4
+    name: commcarehq_p4
+    shards: [615, 819]
+  - django_alias: p5
+    name: commcarehq_p5
+    shards: [820, 1023]
```

Comment:
@snopoke done. also explicitly set database for formplayer and UCRs. and enabled pg_query_stats for UCRs @mkangia